.github/workflows/swift_package_test.yml

```diff
@@ -175,6 +199,82 @@ jobs:
       - name: Build / Test
         run: ${{ inputs.linux_build_command }} ${{ (contains(matrix.swift_version, 'nightly') && inputs.swift_nightly_flags) || inputs.swift_flags }}
 
+  linux-static-sdk-build:
+    name: Linux Static SDK Build (${{ matrix.swift_version }} - ${{ matrix.os_version }})
+    if: ${{ inputs.enable_linux_static_sdk_build }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        swift_version: ${{ fromJson(inputs.linux_static_sdk_versions) }}
+        os_version: ${{ fromJson(inputs.linux_os_versions) }}
+    container:
+      image: ${{ (contains(matrix.swift_version, 'nightly') && 'swiftlang/swift') || 'swift' }}:${{ matrix.swift_version }}-${{ matrix.os_version }}
+    steps:
+      - name: Swift version
+        run: swift --version
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Provide token
+        if: ${{ inputs.needs_token }}
+        run: |
+            echo "GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}" >> $GITHUB_ENV
+      - name: Set environment variables
+        if: ${{ inputs.linux_env_vars }}
+        run: |
+            for i in "${{ inputs.linux_env_vars }}"
+            do
+              printf "%s\n" $i >> $GITHUB_ENV
+            done
+      - name: Pre-build
+        run: ${{ inputs.linux_pre_build_command }}
+      - name: Install static SDK and build
+        env:
+          BUILD_FLAGS: ${{ (contains(matrix.swift_version, 'nightly') && inputs.swift_nightly_flags) || inputs.swift_flags }}
+        run: |
+          ${{ inputs.linux_static_sdk_pre_build_command }}
+          which curl || (apt -q update && apt -yq install curl)
+          curl -s --retry 3 https://raw.githubusercontent.com/jrflat/github-workflows/refs/heads/linux-static-sdk-build/.github/workflows/scripts/install-and-build-with-sdk.sh | \
```

Comment:
Will change this to `swiftlang/github-workflows/refs/heads/main` before merging, but I wanted to get testing in this PR.

Reply:
Updated in https://github.com/swiftlang/github-workflows/pull/142/commits/85552f942bc521fb3ff77694035589342c9da09d